web/modules/custom/brms_sor/brms_sor.module

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * @file
+ * This is the BRMS System of Record module for sharing data across NFA sites.
+ */
+
+use Drupal\node\NodeInterface;
+
+/**
+ * Implements hook_ENTITY_TYPE_update() for node entities.
+ */
+function brms_sor_node_update(NodeInterface $node) {
+  if ($node->getType() == 'forest_reserve') {
+    $service = \Drupal::service('brms_sor.forest_reserve_service');
```

Comment:
Let's qualify this name rather than use $service

Reply:
Probably good if you type hint it as well

Reply:
Done